src/engraving/libmscore/slur.cpp

```diff
@@ -473,6 +473,9 @@ void SlurSegment::avoidCollisions(PointF& pp1, PointF& p2, PointF& p3, PointF& p
         }
         // Check collisions
         for (Segment* seg : segList) {
+            if (!seg->enabled()) {
+                continue;
+            }
```

Comment:
It might be a little bit more efficient to filter out disabled segments directly when computing `segList`. How about that?

Reply:
right, good point!